packages/ui/src/components/CopyButton/index.tsx

```diff
@@ -16,7 +17,7 @@ type CopyButtonProps = {
   bordered?: boolean
   className?: string
   'data-testid'?: string
-  children?: string
+  children?: ReactNode
```

Comment:
This was done on purpose to be a string, why do you need a react node ? We don't want the devs to be able to put anything else than a string

Reply:
Is it because of translation?